node_modules/oae-core/signin/signin.html

```diff
@@ -1,81 +1,126 @@
+					<li>
+						${authExternalButton(strategy, externalAuthOpts)}
+					</li>
+				{/for}
+			</ul>
+		</div>
+		{if authStrategyInfo.hasLocalAuth}
+			<div id="signin-options-divider">
+				<div class="oae-hr">
+					<span>__MSG__OR__</span>
+				</div>
+			</div>
+		{/if}
+	{/if}
+	{if authStrategyInfo.hasLocalAuth}
+		<div id="signin-options-local">
+			<form id="signin-local-form" role="form">
+				<div class="form-group">
+					<label for="signin-local-username" class="control-label">__MSG__USERNAME__</label>
+					<input id="signin-local-username" name="username" type="text" autocapitalize="off" autocorrect="off" class="form-control required" maxlength="255" placeholder="__MSG__ENTER_YOUR_USERNAME__" />
+				</div>
+				<div class="form-group">
+					<label for="signin-local-password" class="control-label">__MSG__PASSWORD__</label>
+					<input id="signin-local-password" name="password" type="password" class="form-control required" maxlength="255" placeholder="&#8226;&#8226;&#8226;&#8226;&#8226;&#8226;&#8226;" />
+				</div>
+				<div class="form-group">
+					<button type="submit" class="btn btn-primary">__MSG__SIGN_IN__</button>
+				</div>
+			</form>
+		</div>
+	{/if}
+	</ul>
+--></div>
+
+<div id="select-other-tenancy-template"><!--
+    <div id="signin-options-divider">
+        <div class="oae-hr">
+            <span>__MSG__OR__</span>
         </div>
-        {if authStrategyInfo.hasLocalAuth}
-            <div id="signin-options-divider">
-                <div class="oae-hr">
-                    <span>__MSG__OR__</span>
-                </div>
-            </div>
-        {/if}
-    {/if}
-    {if authStrategyInfo.hasLocalAuth}
-        <div id="signin-options-local">
-            <form id="signin-local-form" role="form">
-                <div class="form-group">
-                    <label for="signin-local-username" class="control-label">__MSG__USERNAME__</label>
-                    <input id="signin-local-username" name="username" type="text" autocapitalize="off" autocorrect="off" class="form-control required" maxlength="255" placeholder="__MSG__ENTER_YOUR_USERNAME__" />
-                </div>
-                <div class="form-group">
-                    <label for="signin-local-password" class="control-label">__MSG__PASSWORD__</label>
-                    <input id="signin-local-password" name="password" type="password" class="form-control required" maxlength="255" placeholder="&#8226;&#8226;&#8226;&#8226;&#8226;&#8226;&#8226;" />
-                </div>
-                <div class="form-group">
-                    <button type="submit" class="btn btn-primary">__MSG__SIGN_IN__</button>
-                </div>
-            </form>
+    </div>
+	<h4 id="signin-modal-title">__MSG__SIGN_IN_TO_OTHER_TENANCY__</h4>
+    <div id="tenancyForm">
+        <select tabindex="2" id="tenancyInput">
+            <option value="3620194" selected="selected"> - Type your institution -</option>
```

Comment:
Is there a reason for this specific option value? I'd prefer using something like `---` instead for the empty default.

Reply:
Copy paste ✂️